src/components/VmUserMessages/index.js

```diff
@@ -85,10 +85,12 @@ class VmUserMessages extends React.Component {
         <NotificationDrawer hide={!this.state.show} expanded={this.state.expanded}>
           <NotificationDrawer.Title onCloseClick={this.handleToggle} onExpandClick={this.handleExpand} />
           <NotificationDrawer.PanelBody className={style['panel-body']}>
-            {messagesList}
+            <div className={style['drawer-pf-notifications-list']}>
```

Comment:
```suggestion
            <div className={style['notifications-list']}>
```
Shorter class names with this kind of css import.